### DISCUSSION TITLE
bun.sh redesign

### DISCUSSION THREAD

**Discussion by Murtatrxx** [oven-sh/bun#388] [Ideas]
Currently bun’s website has nothing looks good to eye and very static. I could design 3d fully animated website (following the theme). What do you guys think? Do this project need a good website and docs?

**Comment by dulranga** [comment]
Yes this project definitely needs good accessible documentation and landing page

**Comment by lucasmichot** [comment]
cf. https://github.com/Jarred-Sumner/bun/discussions/354

**Comment by corysimmons** [comment]
@Murtatrxx I can help. Maybe you focus on the sweet landing page with Three.js (I can help with that too if you'd like), and I can work on either custom docs or use Docusaurus?

Should we confirm with Jarred (who is probably drowning in success right now) that this is something he'd be interested in, or just start?

**Reply by dulranga** [reply]
yes let's ask what @Jarred-Sumner thinks about this.

I'm also willing to contribute to create a nice documentation for this

**Reply by lucasmichot** [reply]
Count me in too

**Comment by Murtatrxx** [comment]
It’s so amazing to hear these. My discord is ImRopoxPro#0008 (also I’m on the top of the server you can see me there when I’m online) Please contact me maybe we can create group or something like that.

**Reply by dulranga** [reply]
sent a request

**Reply by 0xrishavsharma** [reply]
> It’s so amazing to hear these. My discord is ImRopoxPro#0008 (also I’m on the top of the server you can see me there when I’m online) Please contact me maybe we can create group or something like that.

Count me in!
I'll be happy to improve and add sections in the website.

**Reply by afkcodes** [reply]
Hey @Murtatrxx will be happy to contribute, im in

**Reply by Murtatrxx** [reply]
@afkcodes You can add me on Discord

**Reply by afkcodes** [reply]
sure, sent a request please check

**Comment by santosant** [comment]
In this discussion could we talk about Docs on the landing page as well? The site is still redirecting to Github

**Reply by Murtatrxx** [reply]
Yes, of course you can add me if you want to join
